apps/codeforafrica/src/lib/data/rest/index.js

```diff
@@ -36,6 +36,13 @@ export const api = {
 
 export async function getPageStaticProps(context) {
   const props = await getPageProps(api, context);
+  if (!props) {
+    return {
+      props: {
+        title: "Page not found",
+      },
+    };
+  }
```

Comment:
If we're trying to provide default values for 404 page, we can do that in the 404 page. Other pages should just set [`notFound`](https://nextjs.org/docs/pages/api-reference/functions/get-static-props#notfound).